### PR TITLE
refactor pages, templates and components

### PR DESCRIPTION
**Pull request by cbuv**
The main reason for this change is to cleanup the dependency mess, and to encourage new developers to contribute, with a easier to navigate codebase. 

* Smart components: pages and templates (page-query)
     * local state: e.g tags and search query. 
* Dumb components:
     * components (coupled to one smart container) 
     * ui-components (shared). 

Ideally, the code in pages/templates subfolders should be minimized, reused where possible and moved to components. 

Signed-off-by: cbuv <cbuv@equinor.com>

### CODE REVIEW

**Comment by eoaksnes** [comment]
Maybe skip /templates/blog-template to /templates/blog etc.?

**Comment by cbuv** [comment]
> Maybe skip /templates/blog-template to /templates/blog etc.?

fixed in 2d9fabb

File: src/components/BlogListing.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import React from "react";
 import { Link } from "gatsby";
 import Img from "gatsby-image";
 import styled from "styled-components";
-import { Authors } from "../components/Bio";
+import { Authors } from ".";
```
[the remark this answers is not in the document]
Comment (cbuv) [reply src/components/BlogListing.js]: using index internally has the benefit that changes to Authors or renaming Bio wont affect anything outside of the component package. Suggest keeping it

File: src/components/layout/Layout.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import React from "react";
-import GlobalStyle from "../ui/global-style";
-import EquinorLogo from "../assets/equinor.png";
+import GlobalStyle from "../../ui/global-style";
```
[the remark this answers is not in the document]
Comment (cbuv) [reply src/components/layout/Layout.js]: fixed in e2dde05

File: src/components/Card.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import styled from "styled-components";
-import media from "../media-query";
+import media from "../ui/media-query";
```
[the remark this answers is not in the document]
Comment (cbuv) [reply src/components/Card.js]: fixed in e2dde05

**Pull request merged**